src/main/java/com/synopsys/integration/jenkins/scan/service/scm/BitbucketRepositoryService.java

```diff
@@ -36,9 +36,9 @@ public Bitbucket fetchBitbucketRepositoryDetails(Jenkins jenkins, Map<String, Ob
                 BitbucketSCMSource bitbucketSCMSource = (BitbucketSCMSource) scmSource;
 
                 String bitBucketToken = (String) scanParameters.get("bitbucket_token");
-                if(bitBucketToken == null) {
-                    bitBucketToken = SCMRepositoryService.getCredentialsToken(bitbucketSCMSource.getCredentialsId());
-                }
+//                if(bitBucketToken == null) {
+//                    bitBucketToken = SCMRepositoryService.getCredentialsToken(bitbucketSCMSource.getCredentialsId());
+//                }
```

Comment:
If we don't get bitbucketToken either from pipeline input or global config but prComment is set to true then, 
there should be some error for user.